after/syntax/lua.vim

```diff
@@ -0,0 +1,6 @@
+if dracula#should_abort('lua')
+    finish
+endif
+
+hi! link luaFunc DraculaCyan
+hi! link luaTable DraculaGreen
```

Comment:
```suggestion
hi! link luaFunc  Function
hi! link luaTable DraculaFg
```

Reply:
In the spec it says [built ins should be cyan](https://spec.draculatheme.com/#sec-Language-Built-ins) unless I'm misreading it. Function declarations (with the function keyword) are green from the well named and completely distinct from "luaFunc", luaFunction highlight group.